### ISSUE TITLE
Remove deprecated t.r.procmon.ProcessMonitor.active

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5517]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5517|
|Type|task|
|Created|2012-03-07 20:23:18Z|
|Branch|https://github.com/twisted/twisted/tree/remove-deprecated-5517|

`ProcessMonitor.active`, `consistencyDelay`, and `consistency` in [twisted/runner/procmon.py](twisted/runner/procmon.py) are deprecated since 10.1 and can be removed.

```
twisted.runner.test.test_procmon.ProcmonTests.test_activeAttributeEqualsRunning ... [OK]
/home/buildbot/twisted/slave.trunk/fedora11-64bit-py2.7/Twisted/twisted/runner/test/test_procmon.py:484:
DeprecationWarning: active is deprecated since Twisted 10.1.0.  Use running instead.
```

Attachments:

* [5517.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/fde/fdec3fdd921dbd34534306167a1961522325ac2c/4c85f067bd8a64b0f478b619fb0830277317bcb5.diff) (3472 bytes) - added by candre717 on 2012-03-12 05:59:14Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5517 5517
type__task task
reporter__thijs thijs
priority__low low
milestone__ 
branch__branches_remove_deprecated_5517 branches/remove-deprecated-5517
branch_author__habnabit habnabit
status__closed closed
resolution__fixed fixed
component__runner runner
keywords__easy easy
time__1331151798000000 1331151798000000
changetime__1331576061000000 1331576061000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="candre717's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|candre717 set owner to candre717|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="candre717's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|candre717 removed owner|
|-|-|

Attached is a patch for this issue, and I've confirmed that I ran a test suite and nothing broke from the changes.

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)<a name="note_4"></a>|@habnabit commented|
|-|-|

(In [33684]) Branching to 'remove-deprecated-5517'

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)<a name="note_5"></a>|@habnabit commented|
|-|-|

(In [33685]) Adding patch from ticket.

Refs [#5517](https://github.com/twisted/twisted/issues/5517).

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra commented|
|-|-|

The news file should go in `twisted/runner/topfiles`.

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)<a name="note_7"></a>|@habnabit commented|
|-|-|

Fix looks good; merging.

**Comment by twisted-trac** [comment]
|[<img alt="habnabit's avatar" src="https://avatars.githubusercontent.com/u/21168?s=50" width="50" height="50">](https://github.com/habnabit)<a name="note_8"></a>|@habnabit set status to `closed`|
|-|-|

(In [33690]) Merge remove-deprecated-5517: Remove deprecated t.r.procmon.ProcessMonitor.{active,consistencyDelay,consistency}.

Author: candre717
Reviewer: habnabit
Fixes: [#5517](https://github.com/twisted/twisted/issues/5517)

ProcessMonitor.active, consistencyDelay, and consistency in twisted.runner.procmon were deprecated since 10.1 have been removed.
